modules/administration-guide/partials/proc_restoring-a-backed-up-instance-of-an-earlier-version-of-che.adoc

```diff
@@ -0,0 +1,49 @@
+
+[id="restoring-a-backed-up-instance-of-an-earlier-version-of-{prod-id-short}_{context}"]
+= Restoring a backed up instance of an earlier version of {prod-short}
+
+You can restore a {prod-short} instance from a backup snapshot that was taken in an earlier version of {prod-short}, and you can roll back {prod-short} to that earlier version.
```

Comment:
This also works with restoring using backup custom resource. Each backup custom rsource has version of Che. So, if current is not the same as in the backup custom resource, chectl will change the version of Che too.